src/components/inputs/ChipInput.jsx

```diff
@@ -358,6 +371,27 @@ const ChipInput = React.forwardRef(function ChipInputFn(
 	);
 
 	const onFocus = useCallback(() => setIsActive(true), []);
+
+	const onPaste = useCallback(
+		(e) => {
+			e.preventDefault();
```

Comment:
mmm, now you are always preventing paste, even on chipInputs that are not handling the custom behaviour, so text is not pasted even in those cases. I think it's better to prevent default behaviour only when the custom behaviour is wanted, or a user might think the component is broken if text is not shown

Reply:
Shit :( ,I have fixed it now and thanks